### DISCUSSION TITLE
モバイル学習コンテンツでcurlコマンドでエラーになる

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#255] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/networking/api-request

REST API呼び出し時、記載されているcurlコマンドを実行するとエラーになる。（コマンドプロンプト/PowerShell）
`curl http://localhost:9080/api/todos -X POST -d '{"text": "アプリ外からの登録"}' -H "Content-Type: application/json"
`
以下のように修正したら正常にコマンド実行できた。
`curl http://localhost:9080/api/todos -X POST -d "{\"text\": \"アプリ外からの登録\"}" -H "Content-Type: application/json"
`

**Comment by nagano-ryo** [comment]
windows環境の場合に起きると思われる。
https://stackoverflow.com/questions/54272167/curl-3-unmatched-brace-in-url-position-1

**Reply by nagano-ryo** [reply]
🖋: REST API呼び出しのcurlコマンドを修正
#268

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
